content/docs/developers/_index.md

```diff
@@ -0,0 +1,145 @@
++++
+title = "Developers"
+weight = 300
+[extra]
+emoji = "⚒️"
+tile = "I want to develop a client, bridge or bot"
+last_updated = "2022-10-18T16:00:00Z"
++++
+
+## Elements of Matrix
+
+### Homeserver
+
+A homeserver is a piece of software hosting accounts of Matrix users. It is
+bound to a single domain that cannot change over time. The accounts on a server
+have an identifier made of a local part, and a server part. A typical identifier
+would be
+
+```txt
+@username:example.com
+```
+
+Users on a server can send `events` into `rooms`. An event is a particular json
+object, describing what a user is trying to do (join a room, send a message,
+update a specific value…). In the case of instant messaging, rooms are very
+similar to Slack, Discord or IRC channels. Most of the events in such rooms are
+about sending messages. Rooms have a unique technical identifier, and zero or
```

Comment:
```suggestion
messages. Rooms have a unique technical identifier, and zero or
```

Reply:
Hm unsure about this one. Are the events themselves sending messages? Technically, the events _are_ the messages, aren't they?

Reply:
> Technically, the events *are* the messages, aren't they?

Yes, that is what this say if you remove those two words as suggested, no?

Reply:
Ok I really need another coffee, for some reason I didn't see the "sending" was removed too.